eng/common/scripts/Git-Repo-Sync.ps1

```diff
@@ -0,0 +1,119 @@
+ #!/usr/bin/env pwsh -c
```

Comment:
We probably should put this under `eng\scripts` and not `eng\common\scripts` as we `eng\common` syncs to all our repos which I don't think we want for this one.

Reply:
The file directory has been transferred